README.md

```diff
@@ -118,6 +119,10 @@
 - [vweb](https://github.com/vlang/v/tree/master/vlib/vweb) - V's built-in web framework. Used by Vorum.
 - [v-jsonrpc](https://github.com/nedpals/v-jsonrpc) - Basic JSON-RPC 2.0-compliant server written on V.
 
+### Database clients
+
+- [redis](https://github.com/patrickpissurno/vredis) - Redis client for V, written in V.
```

Comment:
Shouldn't the name be `vredis`?

Reply:
Well yes, but actually no.

The thing is, once I finish the core implementation, it would be nice if we could move this repo from patrickpissurno/vredis into vlang/redis.

I can change the readme to vredis for the time being. I’m just concerned about wether the future change to vlang/redis might be confusing to some people.

What do you think?

Reply:
I'm fine with a top level name `redis` (`vpm install redis`, `import redis`)

Can be also moved to vlang/redis in the future